Core/src/org/sleuthkit/autopsy/corecomponents/DataContentViewerMedia.java

```diff
@@ -185,51 +189,59 @@ public void resetComponent() {
      * @return True if a video file that can be displayed 
      */
     private boolean isVideoSupported(AbstractFile file) {
-        String name = file.getName().toLowerCase();
-        
-        if ((containsExt(name, AUDIO_EXTENSIONS) || containsExt(name, videoExtensions)) && 
-                (!videoMimes.isEmpty() && file.isMimeType(videoMimes) == MimeMatchEnum.TRUE)) {
-            return true;
+        if (videoMimes == null || videoMimes.isEmpty()) {
+            return false;
         }
-        return false;
+        try {
+            // check BB
+            ArrayList<BlackboardAttribute> attributes = file.getGenInfoAttributes(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_FILE_TYPE_SIG);
```

Comment:
isVideoSupported and isImageSupported look almost identical, could they delegate to a more general isfileType supported that would take a list of mimes and a list of extensions as well as the AbstractFile?